rpcs3/Gui/SettingsDialog.cpp

```diff
@@ -91,6 +92,7 @@ std::vector<std::string> GetAdapters()
 SettingsDialog::SettingsDialog(wxWindow *parent, rpcs3::config_t* cfg)
 	: wxDialog(parent, wxID_ANY, "Settings", wxDefaultPosition)
 {
+	SetIcon(wxGetApp().m_MainFrame->GetIcon());
```

Comment:
Settings is a modal dialog so it doesn't need icon. That's general convention, at least on windows (look at other software like pcsx2 or dolphin), Anyway, let's not include "controversial" icon changes in this PR (for it to have any chance to be accepted).